### ISSUE TITLE
Issues with the [feature] substitution tag and includes

### ISSUE THREAD

**Issue by kungfujohnjon** [MapServer/MapServer#6113]
The feature substitution tag [(from RFC 36)](https://mapserver.org/output/template_output.html) isn't parsed when it's inside an include.
e.g.
main.tmpl
```
// mapserver template
[
[resultset layer=xlayer][include src="itemlist.tmpl"][/resultset]
]
```

itemlist.tmpl
```
// mapserver template
[feature trimlast=","]{"something":"[item name="whatever"]"},[/feature]
```
For this template, main.tmpl will include itemlist.tmpl, but doesn't parse any of the substitution tags inside it. The output will look like
```
[
[feature trimlast=","]{"something":"[item name="whatever"]"},[/feature]
]
```

Also, the trimlast option for features doesn't work with includes.
e.g.
main.tmpl
```
// mapserver template
[
[resultset layer=xlayer]
[feature trimlast=","][include src="itemlist.tmpl"][/feature]
[/resultset]
]
```
itemlist.tmpl
```
//mapserver template
{"something":"[item name="whatever"]"},
```

In this case, the item tag substitution inside the include does work, but the last comma never gets trimmed, e.g.
```
[
{"something":"data"},
{"something":"data"},
...
{"something":"data"},
]
```

By contrast, if main.tmpl is just:
```
// mapserver template
[
[resultset layer=xlayer]
[feature trimlast=","]{"something":"[item name="whatever"]"},[/feature]
[/resultset]
]
```
then the output is trimmed as expected.

I'm testing this on MapServer 7.6.0 built from source on RHEL 7.

Just for completeness, this would be the bones of a map file that would call this template.
Map File:
```
MAP
...
OUTPUTFORMAT
  NAME "json"
  DRIVER "TEMPLATE"
  MIMETYPE "application/json"
  FORMATOPTION "FILE=/path/to/main.tmpl"
END
...
LAYER
  NAME "xlayer"
  TYPE LINE
  STATUS ON
  INCLUDE "DatabaseConnection.map"
  DATA "gis_geometry FROM (SELECT gis_geometry, uid, whatever FROM x  ) as foo using unique uid using SRID=4326"
  METADATA
    ...
    "wms_include_items"  "uid,whatever"
    "wms_feature_info_mime_type" "application/json"
    ...
  END
  TEMPLATE "json"
  ...
END
END
```

**Comment by jmckenna** [comment]
@sdlime are you tackling this for 8.0?  This could be an important fix I feel.  Please add this to your plate :)

**Comment by sdlime** [comment]
Will try... An increasingly full plate. *sigh*

**Comment by sdlime** [comment]
Just being realistic...

**Comment by geographika** [comment]
Fixed with #7020
